src/Collapsible/index.jsx

```diff
@@ -61,11 +62,21 @@ class Collapsible extends React.Component {
     this.props.onToggle(isOpen);
   }
 
+  renderIcon() {
+    const { icons } = this.props;
+    const { isOpen } = this.state;
+
+    if (icons) {
+      return isOpen ? icons.expanded : icons.collapsed;
+    }
+
+    return <FontAwesomeIcon icon={isOpen ? faAngleUp : faAngleDown} />;
+  }
+
   render() {
     const {
       children,
       expandedTitle,
-      iconId,
```

Comment:
If we remove this `iconId`, is it possible this element may no longer be referenced in the same way as before? As in, is there any harm in leaving the ID just in case there are other pieces of code pointing to this element by ID?

I'm guessing probably not, but just something that came to mind

Reply:
@christopappas Good question. Previously, this component was using Paragon's `Icon` component which accepts the optional `id` prop. If the `id` prop was not supplied, it'd generate a new one with the `newId()` utility function. However, if I remember correctly, we were experiencing an issue where the generated id's were not unique so support for providing an `iconId` instead was added here.

Now that the `Collapsible` component is no longer using the `Icon` component (instead, using `FontAwesomeIcon` fo SVG icons), I think it's safe to remove here.

Reply:
If someone was using the icon ID to get a reference to the icon outside this component, then this will break.  I think we generally agreed a month or two back that that seems really unlikely.  If we wanted to be very safe, we'd mark this build as having a BREAKING CHANGE.